mobile/lib/shared/services/asset.service.dart

```diff
@@ -106,6 +211,128 @@ class AssetService {
     }
   }
 
+  Future<Asset> loadExif(Asset a) async {
+    a.exifInfo ??= await _db.exifInfos.get(a.id);
+    if (a.exifInfo?.iso == null) {
```

Comment:
Why only ISO value is checked here?

Reply:
yeah, this seems weird. exifInfo might be not null but only few values filled (from local assets). In that case, the full remote exifInfo should be loaded. However, the exifInfo values are all optional. So checking for every value would result in always loading the data from the server again and again (because e.g. country stays null all the time).

Best would be to use another `updatedAt` value in the ExifInfo and ExifResponseDto as well